airgun/entities/computeprofile.py

```diff
@@ -0,0 +1,63 @@
+from navmazing import NavigateToSibling
+
+from airgun.entities.base import BaseEntity
+from airgun.navigation import NavigateStep, navigator
+from airgun.views.computeprofile import (
+    ComputeProfileView,
+    ComputeProfileDetailsView,
+)
+
+
+class ComputeProfileEntity(BaseEntity):
+
+    def create(self, values):
+        view = self.navigate_to(self, 'New')
+        view.fill(values)
+        view.submit.click()
+
+    def search(self, value):
+        view = self.navigate_to(self, 'All')
+        return view.search(value)
+
+    def rename(self, value, new_value):
```

Comment:
That code should be handled by navigation part, not the entity one:
```
+        view = self.navigate_to(self, 'All')
+        view.searchbox.search(value)
+        view.rename.click()
```
So it is more like this:
```
+    def rename(self, old_name, new_name):
+        view = self.navigate_to(self, 'Rename', entity_name=old_name)
+        view.fill(new_name)
+        view.submit.click()

+@navigator.register(ComputeProfileEntity, 'Rename')
+class RenameComputeProfile(NavigateStep):
+    VIEW = ComputeProfileDetailsView
+
+    prerequisite = NavigateToSibling('All')
+
+    def step(self, *args, **kwargs):
          self.parent.search(kwargs.get('entity_name')
+        self.parent.browser.click(self.parent.rename)
```